services/app/src/components/chessBoard/Tile.jsx

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable no-underscore-dangle */
```

Comment:
Lets try to keep file level `eslint-disable` statements to a minimum.
If you need to disable eslint checks for some specific lines, prefer `eslint-disable-next-line`.
In this particular case -have a look at documentation for this rule https://eslint.org/docs/rules/no-underscore-dangle and try to update our global eslint config to allow underscores and the start of method names.